.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
```

Comment:
the latest version of the action is v2. can you please also add a dependabot.yml configuration so we keep the dependencies up to date? and please add our own automerge action. inspire from how the get-jwks build is done for instance